connbeat.full.yml

```diff
@@ -1,24 +1,609 @@
+################### Connbeat Configuration Example #########################
+
+# This file is an example configuration file highlighting only some common options.
+# The heartbeat.full.yml file in the same directory contains all the supported options
+# with detailed comments. You can use it for reference.
+#
+# You can find the full configuration reference here:
+# https://github.com/raboof/connbeat
+
+############################# Connbeat ######################################
+
 connbeat:
   expose_process_info: true
-  expose_cmdline: true
-  expose_environ: false
+  expose_cmdline: false
+  expose_environ: true
   aggregation: 10s
   poll_interval: 2s
-  enable_tcp_diag: false
+  enable_tcp_diag: true
+  enable_docker: false
+  docker_environment: ["PATH"]
+
+#================================ General ======================================
+
+# The name of the shipper that publishes the network data. It can be used to group
+# all the transactions sent by a single shipper in the web interface.
+# If this options is not defined, the hostname is used.
+#name:
+
+# The tags of the shipper are included in their own field with each
+# transaction published. Tags make it easy to group servers by different
+# logical properties.
```

Comment:
Maybe also add default values to this, and others, in the doc. I find it useful when I changed a setting, I still know what the default is a year later.
For tags it is obviously [], but it can also be null (null; not existent in the json).